### ISSUE TITLE
Create infrastructure for the Holesky network

### ISSUE THREAD

**Issue by zah** [status-im/infra-nimbus#152]
Nimbus will run 100,000 validator on the new [Holesky testnet](https://github.com/eth-clients/holesky).

We should plan the addition of at least 30 new servers to our fleet in order to be able to operate such a large number of validators in a heavier-to-process network.

As usual, our validator keys are available here:
https://github.com/status-im/nimbus-private/tree/master/holesky_deposits

**Comment by jakubgs** [comment]
Related PR: https://github.com/eth-clients/holesky/pull/41

**Comment by jakubgs** [comment]
Here's some questions:

* What is the purpose of this testnet?
* What's the exact deadline for this? Or more exactly, when is genesis?
* What are the host requirements? Same as Prater? Smaller? Bigger?
* Any idea how big in terms of storage this testnet will be? Small as Sepolia? Big as Prater/Goerli? Bigger?
* What layout will we want for this fleet? Do we need only linux hosts? Mixed?

**Comment by zah** [comment]
This network is going to replace Prater, so we would like to continue all of the practices that we follow on Prater.
The purpose of the network is to be significantly bigger than mainnet, so performance issues in the clients are discovered there first. This may or may not mean that we need bigger hosts than Prater (we don't have much actual performance numbers yet).

**Comment by zah** [comment]
The planned launch date is Sept/15, 2023, 14:00 UTC .

**Comment by jakubgs** [comment]
The current hosts we use from InnovaHosting are like this:

* `nimbus.mainnet` - Xeon CPU E5-2690 v2 @ 3.00GHz (10 cores)
* `nimbus.prater` - Xeon CPU E5-2667 v3 @ 3.20GHz (8 cores)
* `nimbus.sepolia` - Xeon CPU E5-2667 v3 @ 3.20GHz (8 cores)

All hosts have 64 GB of RAM, and disk layout can be customized.

Do you think any of those processors would fit, or would you want me to ask IH sales about something different?

**Comment by jakubgs** [comment]
I have opened a ticket with Innova Hosting sales team:

>Hello,
>
>We will need 30 servers for a new Ethereum Testnet called [Holesky](https://github.com/eth-clients/holesky), which has genesis on Sept/15, 2023, 14:00 UTC.
>
>Sorry for the short notice but I was only informed about this need recently.
>
>I think the Xeon CPU E5-2667 v3 @ 3.20GHz CPUs will be fine, unless you have something stronger in terms of single core performance available or purchasable in tame before the 15th.
>
>As for storage. Is the 1.6 TB NVMe the upper limit of storage size SSDs you have available? If so then we'll need the same setup as with other hosts. One small SSD for OS(at least 100 GB), and two big SSDs for data. If possible it would be nice if the big ones were 2 TB or bigger, but if not possible 1.6 TB will probably be fine.
>
>Cheers!

https://client.innovahosting.net/viewticket.php?tid=774553&c=wQeslgJE

**Comment by jakubgs** [comment]
Their sales rep responded with:

>We do not have 30 servers at the moment, I think we can activate about 10 servers and the rest of the servers untill octomber 1-st.
>
>I will check once again with our CTO and tell you in 1-2 days how many servers we can activate now and how many days we need to order and receive the rest of the servers.

And I added:

>The testnet genesis is on the 15th of September, and since this is a completely new testnet there is nothing to sync, since it has not yet started. So as long as I can get the hosts on 13th or even 14th I can get them ready for genesis on the 15th.
>
>If all 30 at not possible in time for the 15th then we can probably make due with less, but at least half would be good to have. As for CPUs, any chance for anything with higher single-core performance, or not really?

**Comment by jakubgs** [comment]
We had a bit of back-and-forth:

>We can activate about 10 servers quickly.
>For the other 20 servers we are out of SSD 1.6TB at the moment.
>We will receive the next package of 50x SSD 1.6TB in about one month
>You will have to wait for 1.6TB to activate he next 2o servers or we can make a quick order for 800GB SSD or a few SSD-s of 3.84 TB

I said that 800 GB is fine, so they said:

>OK, to make sure
>
>Server HP DL360 Gen9 with 2 PSU
>CPU1: E5-2667 v3
>CPU2: No CPU
>RAM:4x 16GB DDR4
>SSD1: 400GB for OS. NO RAID
>SSD2: 800GB (after september 20, we will receive 1.6TB SAS-SSD. We will insert in in the server and you can move the data from 800GB to 1.6TB. 800GB will be removed after you move the data to 1.6TB). NO RAID
>SSD3: 800GB (after september 20, we will receive 1.6TB SAS-SSD. We will insert in in the server and you can move the data from 800GB to 1.6TB. 800GB will be removed after you move the data to 1.6TB) NO RAID
>
>All disks mentioned above are write intensive
>
>We will also receive up to 20x 3.84 SANDISK, READ intensive. Will you need this kind of SSD ?
>
>Tell me if everything here is OK and I will calculate the price.

But then it turns out they actually can do the 1.6 TB ones:

>I have just received an confirmation from our vendor that the SSD can be delivered earlier by 7 days so I guess we can install all servers with 1.6TB untill September 15-th
>
>You will need to decide faster so we could place the order for components

I responded to them via email while on holidays but it turns out they don't receive those responses.

Confirmed today that is fine and that I need a quote as soon as possible.

**Comment by jakubgs** [comment]
Here's the quote:

>Regarding the server configuration that we were discussing in the ticket.
>
>Server HP DL360 Gen9 with 2 PSU
>CPU1: E5-2667 v3
>CPU2: No CPU
>RAM:4x 16GB DDR4
>Raid controller HP P440ar/2GB
>SSD1: 400GB for OS. NO RAID
>SSD2: 1.6 TB no raid configured.
>SSD3: 1.6 TB no raid configured.
>Internet connection 1Gbps
>
>The price for one server is 173Euro/month.
>
>5% discount can be applied if you pay for one year.
>
>Additional 5% discount can be applied if you pay by crypto.

Which adds up to a total of 56052 EUR after discounts.

**Comment by jakubgs** [comment]
I have opened process ID 3968 in Spiff. Blocked on approval from Johannes.

https://prod.mod.spiff.status.im/admin/process-instances/for-me/manage-procurement:procurement:requisition-order-management:request-goods-services/3968

We really need some short links...

**Comment by jakubgs** [comment]
I have asked them to verify if they purchased the SSDs and they did:

>We have ordered the following SSDs:
>
>400GB SAS-SSD HGST (Write Intensive)
>800GB SAS-SSD HGST (Write Intensive)
>3.84TB Toshiba (Read Intensive)
>
>These have been shipped and are expected to arrive in 5 days.
>
>We have also ordered:
>
>1.6TB SAS-SSD HGST (Write Intensive)
>7.68TB SAS-SSD HGST (Write Intensive)
>
>These will be shipped on Monday and we expect to receive them within 7 days after shipping.
>
>As for payment, we are comfortable accepting USDT on either the ERC20 or TRC20 network. We can also accept ETH and Bitcoin, although we prefer USDT.

They have also informed us they prefer a payment in USDT.

**Comment by jakubgs** [comment]
I have requested the update on servers setup and got back:

>The technical team is now installing the servers, hope it will manage to activate all untill september 15

I have informed them that the sooner I can get a few initial servers the sooner I can work on the new setup to get it done in time.

**Comment by jakubgs** [comment]
And looks like `go-ethereum` release with support for Holesky testnet just came out 2 hours ago:

* https://github.com/ethereum/go-ethereum/releases/tag/v1.13.0

**Comment by jakubgs** [comment]
It appears both Erigon and Nethermind are also ready:

* https://github.com/ledgerwatch/erigon/releases/tag/v2.49.0
* https://github.com/NethermindEth/nethermind/releases/tag/1.20.2

**Comment by jakubgs** [comment]
I've upgraded all 3 roles to the versions that support Holesky:

- [`infra-role-geth#53de8de6`](https://github.com/status-im/infra-role-geth/commit/53de8de6) - upgrade from 1.12.2 to 1.13.0
- [`infra-role-erigon#42c607fa`](https://github.com/status-im/infra-role-erigon/commit/42c607fa) - upgrade form 2.47.0 to 2.49.1
- [`infra-role-nethermind#4231bb5b`](https://github.com/status-im/infra-role-nethermind/commit/4231bb5b) - upgrade form 1.20.1 to 1.20.3

Will roll out new Geth to other fleets.

**Comment by jakubgs** [comment]
Also decided to add verison to Consul metadata:

- [`infra-role-geth#a2f2b10b`](https://github.com/status-im/infra-role-geth/commit/a2f2b10b) - consul: add version to service metadata
- [`infra-role-erigon#904da4e0`](https://github.com/status-im/infra-role-erigon/commit/904da4e0) - consul: add version to service metadata
- [`infra-role-nethermind#2725b8c9`](https://github.com/status-im/infra-role-nethermind/commit/2725b8c9) - consul: add version to service metadata

Which should make it easier to track versions across fleets.

**Comment by jakubgs** [comment]
Got an email Dan Popusoi about some servers being close to becoming available:

>Jakub, I am out of office right now and can't reply by ticket.
>
>Today our technical will send you the login credentials to 10-15 servers.
>
>These servers are installed in our new server room. Waiting for our electrician to connect the server room to electricity.

So hopefully we should have half the fleet up and running today. We'll see.

**Comment by jakubgs** [comment]
Looks like we will received servers tomorrow in the morning. Which means we will have little time for setup.
Hopefully there's nothing to sync, so we just need to have the nodes up and running with validators in time.

I have prepared a new layout for this fleet in a branch: https://github.com/status-im/infra-nimbus/tree/holesky-testnet
https://github.com/status-im/infra-nimbus/blob/82e76b9452da1aa3366a31b23bb04b5670a01e3d/ansible/vars/layout/holesky.yml#L2-L184
The general idea is that we have 30 hosts split into 3 groups:

* Hosts `01-10` running Geth, 4 nodes on each host, ~33000 validators in total. Validator clients on `stable` nodes.
* Hosts `11-20` running Erigon, 4 nodes on each host, ~33000 validators in total. Validator clients on `testing` nodes.
* Hosts `21-30` running Nethermind, 4 nodes on each host, ~33000+ validators in total. Validator clients on `unstable` nodes.

The idea is to see behavior of different execution layer nodes across different branches and numbers of validators.
Also, validator clients are enabled for `stable`

**Comment by jakubgs** [comment]
The new hosts have started trickling in:

![image](https://github.com/status-im/infra-nimbus/assets/2212681/39d48668-ebd0-4610-9a0c-4147b8433f40)

**Comment by jakubgs** [comment]
Key resources:

* Block Explorer (EL+CL) https://holesky.beaconcha.in/
* Ethstats: https://ethstats.holesky.ethpandaops.io/
* Homepage: https://holesky.ethpandaops.io/
* Latest Proposal: [Ethereum Magicians - Predictable Ethereum Testnet Lifecycle](https://ethereum-magicians.org/t/proposal-predictable-ethereum-testnet-lifecycle/11575)
* Some Context: [Ethereum Magicians - Paths out of the Goerli supply mess](https://ethereum-magicians.org/t/testnet-workgroup-paths-out-of-the-goerli-supply-mess/11453)

**Comment by jakubgs** [comment]
We got first batch of hosts, but there's some randomness with CPUs:

| Hostname                           | CPU Model                 |
|------------------------------------|---------------------------|
| linux-01.ih-eu-mda1.nimbus.holesky | Xeon E5-2667 v3 @ 3.20GHz |
| linux-02.ih-eu-mda1.nimbus.holesky | Xeon E5-2667 v3 @ 3.20GHz |
| linux-03.ih-eu-mda1.nimbus.holesky | Xeon E5-2698 v4 @ 2.20GHz |
| linux-04.ih-eu-mda1.nimbus.holesky | Xeon E5-2667 v3 @ 3.20GHz |
| linux-05.ih-eu-mda1.nimbus.holesky | Xeon E5-2667 v3 @ 3.20GHz |
| linux-06.ih-eu-mda1.nimbus.holesky | Xeon E5-2667 v3 @ 3.20GHz |
| linux-07.ih-eu-mda1.nimbus.holesky | Xeon E5-2667 v3 @ 3.20GHz |
| linux-08.ih-eu-mda1.nimbus.holesky | Xeon E5-2667 v3 @ 3.20GHz |
| linux-09.ih-eu-mda1.nimbus.holesky | Xeon E5-2667 v3 @ 3.20GHz |
| linux-10.ih-eu-mda1.nimbus.holesky | Xeon E5-2698 v4 @ 2.20GHz |
| linux-11.ih-eu-mda1.nimbus.holesky | Xeon E5-2667 v3 @ 3.20GHz |
| linux-12.ih-eu-mda1.nimbus.holesky | Xeon E5-2667 v3 @ 3.20GHz |
| linux-13.ih-eu-mda1.nimbus.holesky | Xeon E5-2667 v3 @ 3.20GHz |
| linux-14.ih-eu-mda1.nimbus.holesky | Xeon E5-2667 v3 @ 3.20GHz |
| linux-15.ih-eu-mda1.nimbus.holesky | Xeon E5-2690 v4 @ 2.60GHz |

I will ask support about it.

**Comment by jakubgs** [comment]
Major issues with GitHub timeouts resulted in inability to deploy all nodes on time, we were about 50% ready:

![image](https://github.com/status-im/infra-nimbus/assets/2212681/90bca813-6d09-4d5a-9ccf-9bb1ddbbccd4)

The issues were reported to Innova, but not much was improved in time. Lots of timeouts when checking out repos:
```
fatal: unable to access 'https://github.com/status-im/nim-kzg4844.git/': Failed to connect to github.com port 443 after 130761 ms: Connection timed out
```
```
fatal: unable to access 'https://github.com/gnosischain/configs.git/': Failed to connect to github.com port 443 after 129404 ms: Connection timed out
```
```
fatal: unable to access 'https://github.com/eth-clients/eth2-networks.git/': Failed to connect to github.com port 443 after 129505 ms: Connection timed out
```
Which is kinda the same issue we've been having with InnovaHosting MacOS hosts in CI.

**Comment by jakubgs** [comment]
Here's the config changes:

- [`infra-nimbus#08eed943`](https://github.com/status-im/infra-nimbus/commit/08eed943) - holesky.tf: add first 15 hosts for nimbus.holesky
- [`infra-nimbus#f501f820`](https://github.com/status-im/infra-nimbus/commit/f501f820) - nimbus.holesky: fleet config and validator layout

We'll need to re-deploy validators to use the proper layout once we have all 30 hosts.

**Comment by jakubgs** [comment]
Added two more hosts that were bootstrapped:

- [`infra-nimbus#dae14e2a`](https://github.com/status-im/infra-nimbus/commit/dae14e2a) - holesky.tf: add and bootstrap two more hosts

Also fixed installation of Netdata by using their official APT repository:

- [`infra-role-bootstrap-linux#b705392f`](https://github.com/status-im/infra-role-bootstrap-linux/commit/b705392f) - monitor: install Netdata using oficial APT repository

**Comment by jakubgs** [comment]
Also found a bug with new Erigon verison where metric type contains labels:

* https://github.com/ledgerwatch/erigon/issues/8214

Example:
```
admin@linux-05.ih-eu-mda1.nimbus.prater:~ % curl -s http://localhost:7063/debug/metrics/prometheus | grep TYPE | grep rpc
# TYPE rpc_duration_seconds{method="engine_exchangeTransitionConfigurationV1",success="success"} summary
# TYPE rpc_duration_seconds{method="engine_forkchoiceUpdatedV2",success="success"} summary
# TYPE rpc_duration_seconds{method="engine_newPayloadV2",success="success"} summary
# TYPE rpc_duration_seconds{method="eth_syncing",success="success"} summary
# TYPE rpc_failure counter
# TYPE rpc_total counter
```

**Comment by jakubgs** [comment]
I checked up on EL clients and their readiness for new Holesky genesis on the 28th of September 2023, 12:00 UTC and:

* https://github.com/ledgerwatch/erigon/pull/8225
* https://github.com/NethermindEth/nethermind/pull/6112

Nethermind has the update and a release, Erigon has the update but no release, but Go-Ethereum seems to have neither.

**Comment by jakubgs** [comment]
I have bootstrapped the remaining hosts, we have now all 30:

- https://github.com/status-im/infra-nimbus/commit/e40017a2 - holesky.tf: add remaining hosts to the fleet
- https://github.com/status-im/infra-nimbus/commit/39ee763a - nimbus.holesky: switch to correct validators layout

And I'm going to purge all node data on all hosts to start anew.

**Comment by jakubgs** [comment]
I had to use non-official Docker images for Go-Ethereum and Eirgon to support new Holesky genesis:

- [`infra-nimbus#ebb9cc82`](https://github.com/status-im/infra-nimbus/commit/ebb9cc82) - nimbus.holesky: upgrade EL nodes to support new genesis
- [`infra-nimbus#7db4374f`](https://github.com/status-im/infra-nimbus/commit/7db4374f) - nimbus.holesky: drop index from BN and VC names

Deploying now.

**Comment by jakubgs** [comment]
Actually, Erigon just created a release: https://github.com/ledgerwatch/erigon/releases/tag/v2.49.3

- [`infra-nimbus#2588a658`](https://github.com/status-im/infra-nimbus/commit/2588a658) - nimbus.holesky: use 2.49.3 Erigon release

**Comment by jakubgs** [comment]
Looks like Go-Ethereum also had a release 2 hours ago: https://github.com/ethereum/go-ethereum/releases/tag/v1.13.2

- [`infra-nimbus#7b45d24b`](https://github.com/status-im/infra-nimbus/commit/7b45d24b) - nimbus.holesky: upgrade Geth to 1.13.2

Might as well use it.

**Comment by jakubgs** [comment]
I discovered a major issue while checking nodes:
```
admin@linux-01.ih-eu-mda1.nimbus.holesky:/data/beacon-node-holesky-stable % for port in $(seq 9301 9304); do c 0:$port/eth/v1/node/syncing | jq -c; done
{"data":{"head_slot":"0","sync_distance":"0","is_syncing":false,"is_optimistic":true}}
{"data":{"head_slot":"0","sync_distance":"92428","is_syncing":true,"is_optimistic":true,"el_offline":false}}
{"data":{"head_slot":"0","sync_distance":"92428","is_syncing":true,"is_optimistic":false,"el_offline":false}}
{"data":{"head_slot":"0","sync_distance":"92428","is_syncing":true,"is_optimistic":false,"el_offline":false}}
```
Some nodes were showing non-zero `sync_distance`, which is wrong, since this is fresh new genesis.

What turns out is that @zah applied the Holesky fix only to the `stable` branch:

![image](https://github.com/status-im/infra-nimbus/assets/2212681/0b465800-d0d6-48df-99c3-05d2a11cba84)

But `testing`, `unstable`, and `nim-libp2p-auto-bump-unstable` did not include that fix. Made a PR:

* https://github.com/status-im/nimbus-eth2/pull/5464

Based off of the `23.9.1` release commit: https://github.com/status-im/nimbus-eth2/commit/cfa0268d89461bfcba75a6d15e994037a0527726

**Comment by jakubgs** [comment]
I also applied the https://github.com/status-im/nimbus-eth2/commit/cfa0268d89461bfcba75a6d15e994037a0527726 commit directly to the `testing` branch.

**Comment by jakubgs** [comment]
Looks like instead Zahary merged `stable` into `unstable`: https://github.com/status-im/nimbus-eth2/commit/77d6bc5f42015ce2e49aefaaed28bb737a4b84f5

**Comment by jakubgs** [comment]
Some nodes were not online for the Echo 0 because I had to purge data folders for all nodes and then restart, which took ages with Ansible due to endless GitHub timeouts on the hosts:
```
fatal: unable to access 'https://github.com/status-im/nim-blscurve.git/': Failed to connect to github.com port 443 after 130791 ms: Connection timed out
fatal: run_command returned non-zero status for vendor/nim-blscurve
```
Innova Hosting really need to do something about this.

**Comment by jakubgs** [comment]
Also found a bug in ports configuration for EL nodes that I fixed:

- [`infra-nimbus#adc1a061`](https://github.com/status-im/infra-nimbus/commit/adc1a061) - nimbus.holesky: use the same ports for all EL node types

**Comment by jakubgs** [comment]
A set of missed proposals was identified:

| Slot | Host | Node |
|-|-|-|
| https://holesky.beaconcha.in/slot/310 | `linux-20.ih-eu-mda1.nimbus.holesky` | `libp2p` |
| https://holesky.beaconcha.in/slot/314 | `linux-19.ih-eu-mda1.nimbus.holesky` | `stable` |
| https://holesky.beaconcha.in/slot/384 | `linux-20.ih-eu-mda1.nimbus.holesky` | `stable` |
| https://holesky.beaconcha.in/slot/389 | `linux-30.ih-eu-mda1.nimbus.holesky` | `stable` |
| https://holesky.beaconcha.in/slot/390 | `linux-20.ih-eu-mda1.nimbus.holesky` | `stable` |

Predictably they cluster around hosts with most validators.

**Comment by jakubgs** [comment]
All of those misses above most probably are explained by port mismatch that was fixed in https://github.com/status-im/infra-nimbus/commit/adc1a061.

**Comment by jakubgs** [comment]
For some reason Erigon nodes have no peer and refuse to connect to anything at all. Opened an issue about it:

* https://github.com/ledgerwatch/erigon/issues/8320

I tried `static-nodes.json`, `trusted-nodes.json`, and `admin_addPeer` RPC method, but nothing worked so far.

**Comment by jakubgs** [comment]
Found another issue with Erigon, this time it's ignoring `--port` flag and listening on `30303` anyway:

* https://github.com/ledgerwatch/erigon/issues/8330

**Comment by jakubgs** [comment]
I believe this is now done.

**Comment by jakubgs** [comment]
Looks like finally all the servers now have the same CPU: Xeon E5-2667 v3
```
 > a nimbus.holesky --become -o -a 'cat /proc/cpuinfo | grep "model name" | uniq' | sort
linux-01.ih-eu-mda1.nimbus.holesky | CHANGED | rc=0 | (stdout) model name	: Intel(R) Xeon(R) CPU E5-2667 v3 @ 3.20GHz
linux-02.ih-eu-mda1.nimbus.holesky | CHANGED | rc=0 | (stdout) model name	: Intel(R) Xeon(R) CPU E5-2667 v3 @ 3.20GHz
linux-03.ih-eu-mda1.nimbus.holesky | CHANGED | rc=0 | (stdout) model name	: Intel(R) Xeon(R) CPU E5-2667 v3 @ 3.20GHz
linux-04.ih-eu-mda1.nimbus.holesky | CHANGED | rc=0 | (stdout) model name	: Intel(R) Xeon(R) CPU E5-2667 v3 @ 3.20GHz
linux-05.ih-eu-mda1.nimbus.holesky | CHANGED | rc=0 | (stdout) model name	: Intel(R) Xeon(R) CPU E5-2667 v3 @ 3.20GHz
linux-06.ih-eu-mda1.nimbus.holesky | CHANGED | rc=0 | (stdout) model name	: Intel(R) Xeon(R) CPU E5-2667 v3 @ 3.20GHz
linux-07.ih-eu-mda1.nimbus.holesky | CHANGED | rc=0 | (stdout) model name	: Intel(R) Xeon(R) CPU E5-2667 v3 @ 3.20GHz
linux-08.ih-eu-mda1.nimbus.holesky | CHANGED | rc=0 | (stdout) model name	: Intel(R) Xeon(R) CPU E5-2667 v3 @ 3.20GHz
linux-09.ih-eu-mda1.nimbus.holesky | CHANGED | rc=0 | (stdout) model name	: Intel(R) Xeon(R) CPU E5-2667 v3 @ 3.20GHz
linux-10.ih-eu-mda1.nimbus.holesky | CHANGED | rc=0 | (stdout) model name	: Intel(R) Xeon(R) CPU E5-2667 v3 @ 3.20GHz
linux-11.ih-eu-mda1.nimbus.holesky | CHANGED | rc=0 | (stdout) model name	: Intel(R) Xeon(R) CPU E5-2667 v3 @ 3.20GHz
linux-12.ih-eu-mda1.nimbus.holesky | CHANGED | rc=0 | (stdout) model name	: Intel(R) Xeon(R) CPU E5-2667 v3 @ 3.20GHz
linux-13.ih-eu-mda1.nimbus.holesky | CHANGED | rc=0 | (stdout) model name	: Intel(R) Xeon(R) CPU E5-2667 v3 @ 3.20GHz
linux-14.ih-eu-mda1.nimbus.holesky | CHANGED | rc=0 | (stdout) model name	: Intel(R) Xeon(R) CPU E5-2667 v3 @ 3.20GHz
linux-15.ih-eu-mda1.nimbus.holesky | CHANGED | rc=0 | (stdout) model name	: Intel(R) Xeon(R) CPU E5-2667 v3 @ 3.20GHz
linux-16.ih-eu-mda1.nimbus.holesky | CHANGED | rc=0 | (stdout) model name	: Intel(R) Xeon(R) CPU E5-2667 v3 @ 3.20GHz
linux-17.ih-eu-mda1.nimbus.holesky | CHANGED | rc=0 | (stdout) model name	: Intel(R) Xeon(R) CPU E5-2667 v3 @ 3.20GHz
linux-18.ih-eu-mda1.nimbus.holesky | CHANGED | rc=0 | (stdout) model name	: Intel(R) Xeon(R) CPU E5-2667 v3 @ 3.20GHz
linux-19.ih-eu-mda1.nimbus.holesky | CHANGED | rc=0 | (stdout) model name	: Intel(R) Xeon(R) CPU E5-2667 v3 @ 3.20GHz
linux-20.ih-eu-mda1.nimbus.holesky | CHANGED | rc=0 | (stdout) model name	: Intel(R) Xeon(R) CPU E5-2667 v3 @ 3.20GHz
linux-21.ih-eu-mda1.nimbus.holesky | CHANGED | rc=0 | (stdout) model name	: Intel(R) Xeon(R) CPU E5-2667 v3 @ 3.20GHz
linux-22.ih-eu-mda1.nimbus.holesky | CHANGED | rc=0 | (stdout) model name	: Intel(R) Xeon(R) CPU E5-2667 v3 @ 3.20GHz
linux-23.ih-eu-mda1.nimbus.holesky | CHANGED | rc=0 | (stdout) model name	: Intel(R) Xeon(R) CPU E5-2667 v3 @ 3.20GHz
linux-24.ih-eu-mda1.nimbus.holesky | CHANGED | rc=0 | (stdout) model name	: Intel(R) Xeon(R) CPU E5-2667 v3 @ 3.20GHz
linux-25.ih-eu-mda1.nimbus.holesky | CHANGED | rc=0 | (stdout) model name	: Intel(R) Xeon(R) CPU E5-2667 v3 @ 3.20GHz
linux-26.ih-eu-mda1.nimbus.holesky | CHANGED | rc=0 | (stdout) model name	: Intel(R) Xeon(R) CPU E5-2667 v3 @ 3.20GHz
linux-27.ih-eu-mda1.nimbus.holesky | CHANGED | rc=0 | (stdout) model name	: Intel(R) Xeon(R) CPU E5-2667 v3 @ 3.20GHz
linux-28.ih-eu-mda1.nimbus.holesky | CHANGED | rc=0 | (stdout) model name	: Intel(R) Xeon(R) CPU E5-2667 v3 @ 3.20GHz
linux-29.ih-eu-mda1.nimbus.holesky | CHANGED | rc=0 | (stdout) model name	: Intel(R) Xeon(R) CPU E5-2667 v3 @ 3.20GHz
linux-30.ih-eu-mda1.nimbus.holesky | CHANGED | rc=0 | (stdout) model name	: Intel(R) Xeon(R) CPU E5-2667 v3 @ 3.20GHz
```
